q2_feature_table/_summarize/summarize_assets/src/controls.js

```diff
@@ -81,8 +91,7 @@ const initializeControls = () => {
     .attr('display', 'inline-block')
     .style('padding-top', '10px')
```

Comment:
If you made the above simplification, you should be able to do `.append('span')` right after the `.style` call.